src/clint.py

```diff
@@ -2600,16 +2600,23 @@ def CheckBraces(filename, clean_lines, linenum, error):
         else:
             func_start_linenum = end_linenum + 1
             while not clean_lines.lines[func_start_linenum] == '{':
-                if not Match(r'^(?:\s*\b(?:FUNC_ATTR|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?)+$',
-                             clean_lines.lines[func_start_linenum]):
+                attrline = Match(r'^(\s*)(?:\b(?:FUNC_ATTR|FUNC_API|REAL_FATTR)_\w+\b(?:\(\d+(, \d+)*\))?\s*)+$',
+                                 clean_lines.lines[func_start_linenum])
+                if attrline:
+                    if len(attrline.group(1)) != 4:
+                        error(filename, func_start_linenum,
+                              'whitespace/indent', 5,
+                              'Function attribute line should be indented '
```

Comment:
“Two space indent[ation]” is one thing, “indent[ed] with two spaces” is another. I actually searched the file and found that “indent[ed] N spaces” occures in comments (lots of occurrences around 2 200’th line), messages normally use “N-space indent” (lines 2 523, 2 880, 2 904, 2 929), have different wording (lines 2 120) or do not mention any exact indentation (lines 2 113, 2 519). Actually that is all I found that is a regular English text by searching for `\<indent`, there are lots of identifiers though.

So you should probably write neither of my variants, choosing “Function attribute line should have 2-space indent” instead.